src/cloudai/workloads/dynamo_mocker/dynamo_mocker.sh

```
@@ -0,0 +1,789 @@
+      --disaggregation-mode)      disaggregation_mode="$2";    shift 2 ;;
+      --prefill-num-nodes)        prefill_num_nodes="$2";      shift 2 ;;
+      --decode-num-nodes)         decode_num_nodes="$2";       shift 2 ;;
+      --prefill-cmd)              prefill_cmd="$2";            shift 2 ;;
+      --decode-cmd)               decode_cmd="$2";             shift 2 ;;
+      --kv-transfer-bandwidth)    kv_transfer_bandwidth="$2";  shift 2 ;;
+      --prefill-initialized-regex)   prefill_initialized_regex="$2";   shift 2 ;;
+      --decode-initialized-regex)    decode_initialized_regex="$2";    shift 2 ;;
+      --benchmark-tool)              benchmark_tool="$2";              shift 2 ;;
+      --engine-*)
+        # Extra dynamo.mocker engine flags forwarded from [cmd_args.engine] extras.
+        engine_extra_args["--${1#--engine-}"]="$2"
```

Shellcheck (0.11.0)

[warning] engine_extra_args appears unused. Verify use (or export if used externally).

(SC2034)